server/storage/players.lua

```diff
@@ -99,6 +100,7 @@ end
 ---@field position vector4
 ---@field metadata PlayerMetadata
 ---@field cid integer
+---@field LastLoggedOut integer
```

Comment:
```suggestion
---@field lastLoggedOut integer
```
Should be camelCase everywhere as well